app/views/search/search_jobs.html.erb

```diff
@@ -23,4 +23,27 @@
   </div>
 </div>
 
-<%= render partial: 'search_footer' unless request.xhr? %>
+<!-- Navigation -->
+<div class="navigation">
+  <div class="row">
+    <div class="small-12 columns">
+      <ul class="small-block-grid-1 medium-block-grid-3 cards small">
+        <li>
+          <a href="https://veterans.linkedin.com/">
+            <h5>LinkedIn</h5>
+          </a>
+        </li>
+        <li>
+          <a href="https://www.surveymonkey.com/r/QSGQRFN">
+            <h5>Coursera</h5>
+          </a>
+        </li>
+        <li>
+          <a href="/employment/job-seekers/create-resume">
+            <h5>Resume Builder</h5>
+          </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
```

Comment:
This is from the content team per the IA. I've made a bigger ask that they include the descriptions across the board, but it may take some time.  

It should display like piano keys which will make it consistent with most of the other pages - so this is intentional but, noted.